modules/private-clusters-setting-api-private.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/configuring-private-cluster.adoc
+
+[id="private-clusters-setting-api-private_{context}"]
+= Restricting the API server to private
+
+After you deploy a cluster to Amazon Web Services (AWS) or Microsoft Azure, you can reconfigure the API server to use only the private zone.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Have access to the web console as a user with `admin` privileges.
+
+.Procedure
+
+. In the web portal or console for AWS or Azure, take the following actions:
+
+.. Locate and delete the external load balancer. The API DNS entry in the private zone already points to the internal load balancer, which uses an identical configuration, so you do not need to modify the internal load balancer.
```

Comment:
For Azure, the public LB is required for outbound connectivity. Instead of deleting the public LB, we need to delete the public api-internal rule.